tests/__mocks/commands/name.ts

```diff
@@ -0,0 +1,8 @@
+import { Command } from "../../../src/type/command";
+
+export class name extends Command {
```

Comment:
This should probably be renamed, such as "normal", to indicate its the normal command mock, without anything changed